Automattic/jetpack
# CLI Tools: update scaffolding, use block.json to register blocks.

Fixes #33173

## Proposed changes:

Let's update our scaffolding to encourage others to use the new nomemclature introduced in #32602.

### Other information:


## Jetpack product discussion

* #32602

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* On your local checkout, run `jetpack docker wp -- jetpack scaffold block "Amazing Rock" --slug="good-music" --description="Rock the best music on your site" --keywords="music, audio, media, four, five"`, then build with `jetpack build plugins/jetpack`, and observe the new block in the block editor.
